clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -584,7 +585,7 @@ bool EmitAssemblyHelper::AddEmitPasses(legacy::PassManager &CodeGenPasses,
                                        raw_pwrite_stream *DwoOS) {
   // Add LibraryInfo.
   std::unique_ptr<TargetLibraryInfoImpl> TLII(
-      llvm::driver::createTLII(TargetTriple, CodeGenOpts.getVecLib()));
+      createTLII(TargetTriple, CodeGenOpts));
```

Comment:
Why are we reverting community change here?

Reply:
Because we want to overwrite the driver `createTLII` and use local version, so that we can update TLII by calling addAltMathFunctionsFromLib .